MARKETProtocol/website
# Adding Route To Video Section

##### Description
 1.Added new route to static.config.js
 2.Created a route wrapper passing the routeData to HomeComponent
 3.Added test case for video expansionon "./video" route navigation

##### Checklist


- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)
New Route is added so it doesn't affect other views

##### Screenshots (Optional) 
Video Section on "./video" route 
![videosection](https://user-images.githubusercontent.com/42678276/44913275-7d873c00-ad4a-11e8-8725-d10c7ce79226.png)


##### Testing
##Test Path
1. Navigate to Home Page and modify the url to "./video" and see that video is expanded and played.
2. Navigate to other sections to enforce it doesn't after other views.
3. Run test cases

##### Refers/Fixes

 Refs: http://github.com/MARKETProtocol/website/issues/242
